.github/workflows/unit_testing.yml

```diff
@@ -0,0 +1,16 @@
+name: Unit Test Before Updating Dev or Production
+on:
+  push:
+    branches:
+      - dev
+      - main
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '16.x'
```

Comment:
@wesdevpro I think we should test it at least on Node.js v18.x, as [v16.x is ending its life](https://github.com/nodejs/release#release-schedule). We may use a build matrix to run tests on different Node.js versions (https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs):
```yml
    strategy:
      matrix:
        node-version: [14.x, 16.x, 18.x, 20.x]
    steps:
      - uses: actions/checkout@v3
      - uses: actions/setup-node@v3
        with:
          node-version: ${{ matrix.node-version }}
```